File: templates/fsharp/xplat/AvaloniaTest.Android/Application.fs
namespace AvaloniaTest.Android

open Android.App
open Android.Content.PM
open Avalonia
open Avalonia.Android
#if (ReactiveUIToolkitChosen)
open ReactiveUI.Avalonia
#endif
open AvaloniaTest

    [<Application>]
type Application(javaReference: nativeint, transfer: Android.Runtime.JniHandleOwnership) = 
    inherit AvaloniaAndroidApplication<App>(javaReference, transfer)

     override _.CustomizeAppBuilder(builder) =
        base.CustomizeAppBuilder(builder)
            .WithInterFont()
#if (ReactiveUIToolkitChosen)
            .UseReactiveUI(fun _ -> ())
#endif

File: templates/csharp/xplat/AvaloniaTest.Android/Application.cs
using Android.App;
using Android.Runtime;
using Avalonia;
using Avalonia.Android;
#if (ReactiveUIToolkitChosen)
using ReactiveUI.Avalonia;
#endif

namespace AvaloniaTest.Android
{
    [Application]
    public class Application : AvaloniaAndroidApplication<App>
    {
        protected Application(nint javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
        {
        }

        protected override AppBuilder CustomizeAppBuilder(AppBuilder builder)
        {
            return base.CustomizeAppBuilder(builder)
#if (CommunityToolkitChosen)
            .WithInterFont();
#else
            .WithInterFont()
            .UseReactiveUI(_ => { });
#endif
        }
    }
}
